plugins/algoltek-usb/meson.build

```diff
@@ -0,0 +1,16 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginAlgoltekUsb"']
```

Comment:
you can leave the gusb check that was there before, it's the right thing to trigger on for a usb plugin (you can't use a USB plugin without gusb)

Reply:
Updated!